master/internal/experiment/bulk_action.go

```diff
@@ -477,7 +478,8 @@ func ArchiveExperiments(ctx context.Context, system *actor.System,
 		Model(&expChecks).
```

Comment:
archive and unarchive have a lot of duplicated code. they could probably use a refactor.